cli/Cargo.toml

```diff
@@ -4,7 +4,7 @@ version = "4.7.0"
 authors = ["Phylum, Inc. <engineering@phylum.io>"]
 license = "GPL-3.0-or-later"
 edition = "2021"
-rust-version = "1.64.0"
+rust-version = "1.65.0"
```

Comment:
Is it okay to have different MSRV values throughout the repo? With this change, there will be three different values for `rust-version` between the four (4) `Cargo.toml` files in the `cli` repository.